### PR TITLE
Update RN50 examples to use nvJPEG random crop decoding

### PR DESCRIPTION
**Pull request by JanuszL**
- adds triangular resampling for PyTorch example as it works on raw ImageNet
- updates TensorFlow, MXNet and PyTorch examples

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

File: docs/examples/mxnet/mxnet-resnet50.ipynb [hunk 1]
```diff
@@ -303,14 +307,14 @@
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 3
+    "version": 2
```
Comment (jantonguirao) [inline docs/examples/mxnet/mxnet-resnet50.ipynb]: revert that?

File: docs/examples/mxnet/mxnet-resnet50.ipynb [hunk 2]
```diff
@@ -303,14 +307,14 @@
...
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.5.2"
+   "pygments_lexer": "ipython2",
```
Comment (jantonguirao) [inline docs/examples/mxnet/mxnet-resnet50.ipynb]: and that?

File: docs/examples/mxnet/mxnet-resnet50.ipynb (hunk 1 above)
Comment (JanuszL) [reply docs/examples/mxnet/mxnet-resnet50.ipynb]: Done

File: docs/examples/mxnet/mxnet-resnet50.ipynb (hunk 2 above)
Comment (JanuszL) [reply docs/examples/mxnet/mxnet-resnet50.ipynb]: Done

**Comment by JanuszL** [comment]
Builds 676061

**Pull request merged**